helm/alfresco-content-services/values.yaml

```diff
@@ -122,7 +122,9 @@ repository:
 apiexplorer:
   ingress:
     path: /api-explorer
-ooiService:
+alfresco-connector-ms365:
+  enabled: false
+# -- Enable/Disable Alfresco Collaboration Connector for Microsoft 365
```

Comment:
doc annotation must be before the node you want to deocument

Reply:
and you want to document the `enabled` field otherwise the helm-docs generated with annotation on non-leaf are very ugly